src/variables/utils/astim.ts

```diff
@@ -0,0 +1,41 @@
+// Libraries
+import {get} from 'lodash'
+
+// Utils
+import {parse} from 'src/external/parser'
+import {findNodes} from 'src/shared/utils/ast'
+
+// Types
+import {File} from 'src/types'
+
+export interface ASTIM {
+  variables: any[]
+  getAST: () => File
+  hasVariable: (v: string) => boolean
+}
+
+const parseAllVariables = (ast: File): any[] => {
+  return findNodes(ast, node => get(node, 'property.type') === 'Identifier')
```

Comment:
Maybe this outputs an empty array?
```suggestion
  return findNodes(ast, node => get(node, 'property.type') === 'Identifier') ?? []
```

Reply:
Also, an we try not using `get` here?